### PR TITLE
Switch to poetry for packaging

### PR DESCRIPTION
**Pull request by jwelch92**
https://github.com/sdispater/poetry

Install deps `poetry install`
build `poetry build`
publish `poetry publish`

### CODE REVIEW

File: poetry.lock [hunk 1]
```diff
@@ -0,0 +1,57 @@
+[[package]]
+category = "main"
+description = "Python datetimes made easy"
+name = "pendulum"
+optional = false
+python-versions = ">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*"
```
Comment (dwelch91) [inline poetry.lock:6]: Only supporting 3.5+ . (using f-strings, etc)

File: poetry.lock [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+category = "main"
+description = "The Olson timezone database for Python."
+name = "pytzdata"
+optional = false
+python-versions = ">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*"
+version = "2018.7"
+
+[[package]]
+category = "main"
+description = "Python 2 and 3 compatibility utilities"
```
Comment (dwelch91) [inline poetry.lock:42]: Yeah, no 2.x support... it's 2018!!! :-)

**Review by dwelch91 — CHANGES_REQUESTED** [review]

**Comment by dwelch91** [comment]
Oh, wait, are those the versions for the dependencies only?
Even still, I don't want six.py included... not needed.
Unless... a dependency needs it... argh. What a pain to still have to deal with 2.x!

File: poetry.lock (hunk 1 above)
Comment (jwelch92) [reply poetry.lock:6]: That's the versions for pendulum!

File: poetry.lock (hunk 2 above)
Comment (jwelch92) [reply poetry.lock:42]: Thats the versions for the deps!

**Pull request merged**